tests/unittest/_torch/test_connector.py

```
@@ -175,12 +175,154 @@ def test():
+    per_step_chains = []
+    with patch(PATCH_TARGET,
+               side_effect=fake_get_stored_block_hashes) as mock_inc:
+        for tokens in token_growths:
+            req.get_tokens.return_value = tokens
+            output = incremental_manager.build_scheduler_output(
+                scheduled_batch, AsyncRequests({}, {}), kv_cache_manager)
+            per_step_chains.append(list(output.cached_requests[0].block_hashes))
+
+    # A fresh manager seeing the same tokens for the first time must build the
+    # same cumulative chain as the incremental one cached up to that step.
+    for tokens, observed in zip(token_growths, per_step_chains):
```

Ruff (0.15.11)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)